.circleci/src/jobs/@web-jobs.yml

```diff
@@ -62,7 +62,7 @@ web-build-demo:
         at: ./
     - run:
         name: build-demo
-        no_output_timeout: 30m
+        git no_output_timeout: 30m
```

Comment:
I'm not sure.I understand this - what does adding `git` here do?

Reply:
it does not... this was a typo

Reply:
but.. it's the only change?

Reply:
@sliptype fixed it https://github.com/AudiusProject/audius-client/pull/3322/files

I missed that